baram/iam_manager.py

```diff
@@ -61,3 +63,23 @@ def list_group_policies(self, user_group_name, max_items=100):
         :return:
         '''
         return self.cli.list_group_policies(GroupName=user_group_name, MaxItems=max_items)
+
+    def get_policies(self):
+        """
+        Lists all policies in IAM
+        :return:
+        """
+        return self.cli.list_policies(MaxItems=1000)['Policies']
```

Comment:
Isn't it better to parameterize max_items?

Reply:
agreed. will reflect it on next commit